build.gradle.kts

```diff
@@ -164,7 +164,13 @@ subprojects {
 
   java {
     toolchain {
+      // Some JDK vendors like Oracle OpenJDK have problems in building trino-connector.
+      // So we will use AMAZON OpenJDK for trino-connector.
```

Comment:
You should describe the detailed issues here, it's too vague.

Reply:
Added